### PR TITLE
[MBL-15332][Teacher] Syllabus in landscape mode on tablets

### PR DESCRIPTION
**Pull request by kristofnemere**
refs: MBL-15332
affects: Teacher
release note: none

Test plan:
- Course navigation should be on the left side, syllabus should be on the right side of the split view
- Phone navigation should be unchanged

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Review by kdeakinstructure — APPROVED** [review]
QA 👍

File: apps/teacher/src/main/java/com/instructure/teacher/features/syllabus/ui/SyllabusView.kt [hunk 1]
```diff
@@ -25,9 +25,11 @@ import com.instructure.canvasapi2.models.CanvasContext
 import com.instructure.canvasapi2.models.Course
 import com.instructure.canvasapi2.models.ScheduleItem
 import com.instructure.canvasapi2.utils.exhaustive
+import com.instructure.interactions.MasterDetailInteractions
```
Comment (tamaskozmer) [inline apps/teacher/src/main/java/com/instructure/teacher/features/syllabus/ui/SyllabusView.kt:28]: This import is unused.

File: apps/teacher/src/main/java/com/instructure/teacher/fragments/CourseBrowserFragment.kt [hunk 2]
```diff
@@ -20,6 +20,7 @@ import android.animation.ObjectAnimator
 import android.content.Intent
 import android.graphics.Color
 import android.net.Uri
+import android.os.Bundle
```
Comment (tamaskozmer) [inline apps/teacher/src/main/java/com/instructure/teacher/fragments/CourseBrowserFragment.kt:23]: I think this is unused as well.

**Review by tamaskozmer — APPROVED** [review]

File: apps/teacher/src/main/java/com/instructure/teacher/features/syllabus/ui/SyllabusView.kt (hunk 1 above)
Comment (kristofnemere) [reply apps/teacher/src/main/java/com/instructure/teacher/features/syllabus/ui/SyllabusView.kt]: Thanks, fixed.

File: apps/teacher/src/main/java/com/instructure/teacher/fragments/CourseBrowserFragment.kt (hunk 2 above)
Comment (kristofnemere) [reply apps/teacher/src/main/java/com/instructure/teacher/fragments/CourseBrowserFragment.kt]: Thanks, fixed.

**Pull request merged**